src/main/scala/com/mozilla/spark/sql/hyperloglog/functions/package.scala

```diff
@@ -23,7 +23,11 @@ package object functions {
   // See algebird-core/src/main/scala/com/twitter/algebird/HyperLogLog.scala#L194-L210
   // E.g. 12 bits corresponds to an error of 0.0163
   def hllCreate(x: String, bits: Int): Array[Byte] = {
-    val monoid = new HyperLogLogMonoid(bits)
-    HyperLogLog.toBytes(monoid.toHLL(x))
+    x match {
+      case null => null
+      case v: String =>
```

Comment:
`x` is defined as a String param - I don't think we need to include the type here. This could be simplified to `case _`, I think.